plugin/db/mysql/restore.go

```diff
@@ -552,8 +608,8 @@ func (driver *Driver) GetBinlogDir() string {
 	return driver.binlogDir
 }
 
-// FetchAllBinlogFiles downloads all binlog files on server to `binlogDir`.
-func (driver *Driver) FetchAllBinlogFiles(ctx context.Context, downloadLatestBinlogFile bool) error {
+// FetchAllBinlogFilesFromMySQL downloads all binlog files on server to `binlogDir`.
```

Comment:
nit: prefer not to change these names unless necessary for future PRs.

Reply:
Reverted.